andela/casablanca-cfh
# #156096152 Integrate Coveralls Code Coverage

#### What does this PR do?

- It set up Coveralls code coverage reporting
- It adds Coveralls code coverage badge to README.md
#### Description of Task to be completed?
Integrate Coveralls code coverage service
#### How should this be manually tested?

- Clone the repo.
- `cd` into the project directory.
- Run `npm install` to install dependencies.
- Add more test cases.
- Push to Github.
- After Travis build finishes, the code coverage should be reported to Coveralls.
- If the value of the code coverage changes, the badge on the repo would also reflect it.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#156096152
#### Screenshots
![screencapture-github-andela-casablanca-cfh-tree-ch-integrate-coveralls-156096152-2018-03-22-15_21_15](https://user-images.githubusercontent.com/26048536/37776035-bb55bee8-2de4-11e8-8361-423bd41dca4d.png)
#### Questions
None
